app/controllers/api/v1/addresses_controller.rb

```diff
@@ -1,14 +1,14 @@
 class Api::V1::AddressesController < ApplicationController
```

Comment:
Tests for API controllers are written and passing.  Waiting for your review @fiestacasey